lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -140,6 +145,70 @@ def test_unicode_user_partitions(self):
         _groups = [group.name for group in self.course.user_partitions[0].groups]
         self.assertEqual(_groups, user_groups)
 
+    def test_content_group_partition(self):
+        """
+        Test that cohort-schemed user partitions are ignored in the
+        grades export.
+        """
+        # Set up a course with 'cohort' and 'random' user partitions.
+        content_group_partition = UserPartition(
```

Comment:
When I was writing this test, I was still in the user-facing/product mindset of naming things. I'm happy to change the variable names (and strings) to reference 'cohort scheme' over 'content group'.